client/src/components/locations.jsx

```diff
@@ -1,15 +1,46 @@
 import React from 'react';
+import axios from 'axios';
+
 
 class Locations extends React.Component {
   constructor() {
     super();
+    this.state = {
+      locations: []
+    }
+    this.getProductById = this.getProductById.bind(this);
+  }
+
+  getProductById() {
+    const randIndex = Math.floor(Math.random() * 100) + 1;
+    axios.get(`/api/locations?product_id=${randIndex}`)
+      .then(response => this.setState({ locations: response.data }))
+      .catch(error => console.log(error))
+  }
+
+  componentDidMount() {
+    this.getProductById();
   }
 
   render() {
+    let display = this.state.locations.map(location => {
+      return <div key={location.id}>
+        <div className="card">
+          <div>
+            <h3>{location.loc_name}</h3>
+          </div>
+          <img src={location.loc_pic} alt="" />
+        </div>
+      </div>
+    })
     return (
-      <h2>Locations</h2>
+      <div>
+        <h2>Where it is made</h2>
+        <br/>
+        {display}
+      </div>
     )
   }
 }
 
-export default Locations
\ No newline at end of file
+export default Locations;
```

Comment:
@gigicupp again,  this should be in a separate pull request that deals only with jsx components or front end material. That being said, great progress! Can't wait to  see more!

Reply:
@masoncpott yes, definitely will be doing smaller PRs.